src/data-table/DataTableBody.tsx

```diff
@@ -38,17 +46,21 @@ const useStyles = makeStyles({
         background: "#E0E0E0",
     },
     disabledRow: {
-        backgroundColor: "#F5DFDF !important",
+        backgroundColor: "#EEEEEE",
```

Comment:
out of curiosity, why this change in the background color for disabled rows?

Reply:
@SferaDev  I guess now we should set the background color from the app `rowConfig`, now that we can, right?

Reply:
Exactly, I set ``grey`` color for disabled as it's the default in material and in the app we can now override the styling as we want.